locale/en_US/manager.xml

```diff
@@ -191,7 +191,7 @@
 	<message key="manager.statistics.reports.defaultReportTemplates">Default report templates</message>
 	<message key="manager.statistics.reports.generateReport">Generate custom report</message>
 	<message key="manager.statistics.reports.advancedOptions">Advanced Options</message>
-	<message key="manager.statistics.reports.advancedOptions.label">Customize columns, filter and ordering options. Changing any of these options might change the expected report template result.</message>
```

Comment:
As far as I could tell, extrasOnDemand didn't used a label and that was the case before this commit. So just cleaning up an unused message key.